lib/reactive_table.js

```diff
@@ -295,15 +296,21 @@ var getDefaultFieldVisibility = function (field) {
     return new ReactiveVar(!field.hidden || (_.isFunction(field.hidden) && !field.hidden()));
 }
 
+var getRowCount = function () {
+    if(this.isServer) {
```

Comment:
this should be this.server - it always runs on the client and this.server indicates whether it's using a server-side publication.